app/jobs/create_asset_job.rb

```diff
@@ -2,10 +2,14 @@
 class CreateAssetJob < CreateWorkJob
   before_perform do |job|
     job.arguments[2].fetch(:uploaded_files, []).each do |id|
-      file = Sufia::UploadedFile.find(id)
-      unless DuplicateUploadVerificationService.unique?(file)
+      @sufia_uploaded_file = Sufia::UploadedFile.find(id)
```

Comment:
why the @ variable?

Reply:
Because I need to use it later in the `after_perform` down below.

Reply:
@RudyOnRails Yes. I'm not certain that instance variables carry over across before and after blocks. You might want to double-check that, unless the tests are already proving they do.